pkgs/development/python-modules/yoto-api/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytz,
+  requests,
+  paho-mqtt,
+}:
+
+buildPythonPackage rec {
+  pname = "yoto-api";
+  version = "1.24.2";
+  format = "setuptools";
```

```suggestion
  pyproject = true;
```

Uses the pep517 builder instead.